langgraph/checkpoint/mysql/__init__.py

```diff
@@ -119,6 +119,7 @@ def setup(self) -> None:
             ):
                 cur.execute(migration)
                 cur.execute(f"INSERT INTO checkpoint_migrations (v) VALUES ({v})")
+                cur.execute("COMMIT")
```

Comment:
In my testing, the version value is not up to date. So when I run setup next time. It will try to excute previouse migration. Previous migration will cause error.
I didn't dive much too. But I found execute 'commit' can make version up to date. So I added.
And now setup() can been call multiple times.

Reply:
Maybe it's caused by `autocommit` . When I use this repo. I read the readme. Noticed that I need autocommit set to True.

But when I use `sqlalchemy import create_engine`. I didn't find `autocommit` setting. After searching, I think the autocommit is set to True by default. So I didn't set `autocommit` in my code. Just use default value.